featurebyte/query_graph/sql/ast/util.py

```diff
@@ -50,7 +50,7 @@ def prepare_binary_op_input_nodes(
     if table_node is None:
         # In this case, the left node is a column from the request data. The right node must be from
         # a feature and has a valid table node.
-        assert right_node.table_node is not None
+        assert right_node.table_node is not None, "Right node must have a table node"
```

Comment:
@YS-L This is the assertion that causes the error if we don't add the validation.